pixiedust/services/stashCloudant.py

```diff
@@ -19,11 +19,20 @@
 import time
 import requests
 import json
+import pixiedust
+from pyspark import SparkContext
 
 CLOUDANT_CONN_TYPE = "cloudant"
-
+myLogger = pixiedust.getLogger(__name__)
 class StashCloudantHandler(Display):
     def doRender(self, handlerId):
+        sc = SparkContext.getOrCreate()
+        config = sc._conf.getAll()
+        
+        sc.stop()
```

Comment:
@DTAIEB sorry didn't realize that there is only one session, from doc http://spark.apache.org/docs/1.6.0/api/python/pyspark.html#pyspark.SparkContext wordings of getOrCreate() is very confusing `Get or instantiate a SparkContext ...` thinking freeing up or stopping is needed. Sorry.